scripts/devnet-algo/algorand_data/config.json

```diff
@@ -1,4 +1,5 @@
 {
+    "MaxAPIResourcesPerAccount": 0,
```

Comment:
Why?

Reply:
This makes balancesOf never need the indexer, and so theoretically more accurate in less time

Reply:
p.s. 0 = infinity.